where-to-travel/src/main/webapp/script.js

```diff
@@ -235,21 +238,22 @@ function addPlacesFromDirection(lat, lng, place_candidates) {
  */
  function filterByDistance(timeObj, listPlaces) {
   return new Promise(function(resolve) {
-    var userLocation = new google.maps.LatLng(home.lat, home.lng);
-    var userDestinations = [];
-    var acceptablePlaces = [];
+    const userLocation = new google.maps.LatLng(home.lat, home.lng);
+    let userDestinations = [];
+    let acceptablePlaces = [];
```

Comment:
Style guide advises against the use of var and instead suggests const and let.

Reply:
done